src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -32,6 +32,12 @@
 import tempfile
 import requests
 
+from .aaz.latest.network.express_route import Create as _ExpressRouteCreate, Update as _ExpressRouteUpdate
```

Comment:
Please move this code block to "from .aaz.latest.network.application_gateway._update import Update as _ApplicationGatewayUpdate" and keep lexicographical order.